### PR TITLE
Added celery logs to volume object

### LINKED ISSUE #237: Volume celery logs should be available on the volume object

### PR DESCRIPTION
**Pull request by chetnieter**
Adds the needed code to add logs to the volume object. Re-enabled the volume tests and added a test case for logging. Fixed some of the errors/failures for the remaining volume tests although that just uncovered other issues with those test cases. The failing tests are currently skipped until #242 is resolved.

This request resolves #237 and makes progress on #242.

### CODE REVIEW

File: girder/cumulus/server/volume.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 from cumulus.constants import VolumeType
 from cumulus.constants import VolumeState
 from cumulus.common.girder import get_task_token, _get_profile
+from cumulus.aws.ec2 import get_ec2_client
```
Comment (cjh1) [inline girder/cumulus/server/volume.py]: Is this import required?

File: girder/cumulus/plugin_tests/volume_test.py [hunk 2]
```diff
@@ -692,3 +707,58 @@ def test_get_status(self, get_ec2_client):
...
+
+        r = self.request('/volumes/%s/log' % str(volume_id), method='GET',
+                         user=self._user)
+        self.assertStatusOk(r)
+        self.assertEqual(len(r.json['log']), 2)
+
+        r = self.request('/volumes/%s/log' % str(volume_id), method='GET',
+                         params={'offset': 1}, user=self._user)
+        self.assertStatusOk(r)
+        self.assertEqual(len(r.json['log']), 1)
```
Comment (cjh1) [inline girder/cumulus/plugin_tests/volume_test.py]: Can we add a test to ensure that the volume log entries appear on the event stream. It should be pretty much like [this test](https://github.com/Kitware/cumulus/blob/master/girder/cumulus/plugin_tests/cluster_test.py#L957-L988)

Comment (cjh1) [inline girder/cumulus/plugin_tests/volume_test.py]: We should also add a test for getting the log of a newly created volume, I see the the following error: 

```
{
  "message": "KeyError: KeyError('log',)",
  "trace": [
    [
      "/opt/hpccloud/girder/girder/api/rest.py",
      489,
      "endpointDecorator",
      "val = fun(self, args, kwargs)"
    ],
    [
      "/opt/hpccloud/girder/girder/api/rest.py",
      1024,
      "GET",
      "return self.handleRoute('GET', path, params)"
    ],
```

I think the issue is that we need to init log to [] [here](https://github.com/Kitware/cumulus/blob/volume-logs/girder/cumulus/server/models/volume.py#L68-L77)

File: girder/cumulus/server/volume.py (hunk 1 above)
Comment (chetnieter) [inline girder/cumulus/server/volume.py]: No. I had a mock decorator for the get_ec2_client that was pointing the volume rest object rather than the aws model. Once I fixed that I was able to remove this unneeded import.

**Comment by chetnieter** [comment]
@cjh1 PTAL - I committed all your suggested changes.

File: girder/cumulus/plugin_tests/volume_test.py [hunk 3]
```diff
@@ -473,11 +482,13 @@ def test_attach_volume(self, get_ec2_client):
             volume_id, self._trad_cluster_id)
         r = self.request(url, method='PUT',
                          type='application/json', body=json.dumps(body),
-                         user=self._cumulus)
+                         user=self._user)
         self.assertStatus(r, 400)
```
Comment (cjh1) [inline girder/cumulus/plugin_tests/volume_test.py]: Just out of interest why the change is user?

**Comment by cjh1** [comment]
LGTM thanks for making the updates

File: girder/cumulus/plugin_tests/volume_test.py (hunk 3 above)
Comment (chetnieter) [inline girder/cumulus/plugin_tests/volume_test.py]: I was getting an error which I traced back to a query to mongo coming back with None. Digging around in mongo I noticed that the aws profiles it was trying to retrieve have the userId associated with the 'user' login rather than the 'cumulus' login. My changes make this consistent with  aws_test.py where all the request are passed _user rather than _cumulus.

Comment (cjh1) [inline girder/cumulus/plugin_tests/volume_test.py]: OK, thanks
